### PR TITLE
nitweb: stars rating system

### PR DESCRIPTION
**Pull request by Morriar**
Add discrete stars rating system in top right corner of each documentation card.

Can be seen here: http://nitweb.moz-code.org/doc/core::Array

### CODE REVIEW

File: src/nitweb.nit [hunk 1]
```diff
@@ -60,14 +60,19 @@ private class NitwebPhase
 			catalog.package_page(mpackage)
 		end
 
+		# Prepare mongo connection
+		var mongo = new MongoClient("mongodb://localhost:27017/")
```
Comment (jcbrinfo) [inline src/nitweb.nit]: Magic number.

**Comment by jcbrinfo** [comment]
I’m unable to change my rating.

**Comment by jcbrinfo** [comment]
I do not understand why the rating bar is hidden by default since:
* It shows the mean.
* There is no visual clue that the user has to hover the card to display it.

**Comment by privat** [comment]
+⭐

**Pull request merged**